default/stringtables/en.txt

```diff
@@ -3194,6 +3194,20 @@ Unable to create persistent_config.xml, check log file for details.
 MAP_BTN_TURN_UPDATE
 Turn %1%
 
+MAP_BTN_TURN_TOOLTIP
+Start Turn
+
+MAP_BTN_TURN_TOOLTIP_DESC
+'''Starts turn processing with the following sequence:
+
+<u>Order execution</u>
+Fleet movement, Colonisation, Invasion, Bombardment etc.
+
+<u>Combat resolution</u>
+
+<u>Production and Research</u>
```

Comment:
break down into more details / steps?

Reply:
Done, open for suggestions concerning the combat phase.